flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerRunnerResult.java

```diff
@@ -81,21 +83,23 @@ public boolean equals(Object o) {
             return false;
         }
         JobManagerRunnerResult that = (JobManagerRunnerResult) o;
-        return Objects.equals(archivedExecutionGraph, that.archivedExecutionGraph)
+        // TODO: verify ExecutionGraphInfo.equals
+        return Objects.equals(executionGraphInfo, that.executionGraphInfo)
                 && Objects.equals(failure, that.failure);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(archivedExecutionGraph, failure);
+        // TODO: verify ExecutionGraphInfo.hashCode
```

Comment:
still valid?

Reply:
See [related discussion](https://github.com/apache/flink/pull/14804#discussion_r580067125) for the corresponding `equals()` method.